pkg/controller/volume/expand/expand_controller.go

```diff
@@ -465,7 +469,7 @@ func (expc *expandController) GetServiceAccountTokenFunc() func(_, _ string, _ *
 
 func (expc *expandController) DeleteServiceAccountTokenFunc() func(types.UID) {
 	return func(types.UID) {
-		klog.ErrorS(nil, "DeleteServiceAccountToken unsupported in expandController")
+		expc.logger.Error(nil, "DeleteServiceAccountToken unsupported in expandController")
```

Comment:
See above.

Reply:
I think we should handle this same as
https://github.com/kubernetes/kubernetes/pull/116930#discussion_r1229517852

Use nolint since this line will never be reached.

Reply:
Done